tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -87,6 +87,18 @@
 #define DAC_RESOLUTION		12
 #define DAC_CHANNEL_ID		0
 
+#elif defined(CONFIG_BOARD_RD_RW612_BGA)
+
+#define DAC_DEVICE_NODE		DT_NODELABEL(dac0)
+#define DAC_RESOLUTION		10
```

Comment:
Can you please point me to the datasheet of this SOC? I couldn't find it.

Does its DAC really only have 10 bits? If it has 12 bits as well, we could move it to the existing group above as well.

Reply:
The data from this DAC is indeed a 10 bit resolution.

This SOC does not have a reference manual or datasheet.